web/ui/react-app/src/pages/graph/GraphHelpers.ts

```diff
@@ -158,14 +159,12 @@ export const getOptions = (stacked: boolean, useLocalTime: boolean): jquery.flot
   };
 };
 
-// This was adapted from Flot's color generation code.
 export const getColors = (data: {
   resultType: string;
   result: Array<{ metric: Metric; values: [number, string][] }>;
 }): Color[] => {
-  const colorPool = ['#edc240', '#afd8f8', '#cb4b4b', '#4da74d', '#9440ed'];
+  const colorPool = ListOfColors;
```

Comment:
We can just use the imported list directly vs. assigning it to a new name here.

Reply:
yup, makes sense. will update it.